STEERING-COMMITTEE.md

```diff
@@ -1,32 +1,115 @@
-3. Control access to, establish processes regarding, and provide a final escalation path for any Istio repository.
-4. Control and delegate access to and establish processes regarding other project resources and assets, including artifact repositories, build and test infrastructure, web sites and their domains, blogs, social-media accounts, etc.
-5. Manage the Istio brand to decide which things can be called “Istio” and how that mark can be used in relation to other efforts or vendors.
-6. Resolve any dispute from the Technical Oversight Committee.
+The Steering Committee’s responsibilities include:
+
+1. Establishing rules of governance for the Istio project, including creation
+and ratification of bylaws for the Steering Committee and Technical Oversight
+Committee.
+1. Establishing and enforcing principles to guide the Istio project.
+1. Fostering an environment for a healthy and happy community of developers,
+contributors, vendors, and users.
+1. Defining, evolving, and defending a
+[Code of Conduct](CONTRIBUTING.md#code-of-conduct).
+1. Advising the Open Usage Commons on issues relating to the Istio trademark and
+logo, as well as related conformance programs.
+1. Setting marketing and advocacy direction for the project; establishing a
+publishing schedule and vetting content, encouraging and assisting community
+members in creating content for conferences, fostering an ecosystem of vendors.
+1. Controlling and delegating access to, and establishing processes regarding,
+project resources/assets, including but not limited to artifact repositories,
+build and test infrastructure, web sites and their domains, blogs, and social
+media accounts.
+1. Providing neutral mediation as appropriate to try to resolve non-technical
+disputes that arise as part of the project.
+
+## Membership and voting
+
+1. A Company shall mean an entity which employs a member, and all other
+entities that control, are controlled by, or are under common control with that
+entity.
+1. The Steering Committee is structured to allow Companies who are most
+invested in the success of the Istio project to participate in business and
+non-technical decision-making.
+1. All members should abide by the project Code of Conduct.
+1. There are two types of seats on the Steering Committee: Contribution Seats
+and Community Seats.
+1. Both Contribution and Community Seats will be appointed beginning in
+August 2020. The appointments for the Contribution and Community seat types will
+expire on staggered dates. After the initial term for both seats, all seats
+will have an annual term.
+1. Contribution Seat terms expire on January 31 and Community Seat terms expire
+on July 31. If necessary, a company holding a Contribution Seat may change the
+appointed individual at any time during the term.
+1. No Company can have more than 5 seats in total on the Steering Committee.
```

Comment:
I think less than 1/3 is resonable.

xref: https://github.com/kubernetes/steering/blob/master/elections.md#maximal-representation
```
the results of an election result in greater than 1/3 representation, the lowest vote getters from any particular company will be removed until representation on the committee is less than one-third.
```

Reply:
And kubernetes charter https://github.com/kubernetes/steering/blob/master/charter.md#quorum
```
Steering committee members are generally expected to attend every meeting. We use the following guidelines to determine whether we have reached quorum and are able to proceed with a meeting.

- Round quorum up if total number of members is odd
- Quorum to meet is 1/2 members attending (eg: 6/12, 6/11, 4/7)
- Quorum to vote in a meeting is 2/3 members attending (eg: 8/12, 8/11, 5/7)
```

Reply:
Good examples.  I would also prefer no one company holds over 1/3, which was in my initial proposal as well.

Reply:
The max number of contribution members of one company should be "total number of contribution member/the number of eligible company".  In case of 9 seats and 3 companies, it should be 9/3 = 3. I would also argue that we may want to open the seats to more companies to encourage wider industry participation.

Reply:
I also agree to use a percentage of voting seats to define the max number of seats versus a flat number. 
I like a percentage of 1/3 (33%) max voting seats but I would place the percentage against the total number of voting seats in the committee and not limit the max to simply the contributor seats. So if the total number is 13 then the max would be 4 seats.

Reply:
It is against the total seats: the percentage is 38.4%, and we're choosing to express it in a number of seats (5). 😄 
33% of 13 is 4.29 seats. Which way do we round it?  A flat number seems easier for that purpose.